Payment UseCase/HotelTest.java

```diff
@@ -2,6 +2,7 @@
 import org.junit.Test;
 
 
+
 //Testing HotelTest class using JUnit test framework
 public class HotelTest {
 //checks if the payment method is correctly "Credit Card" and valid amounts =150

```

Comment:
//you can add here Helper method for testing payment processing and Helper method for testing payment storage

Reply:
for exampel assertPaymentProcessing(String method, double amount)for  testing payment processing